DocumentAi/VERSION

```diff
@@ -1 +1 @@
-2.0.0
+1.5.0
```

Comment:
I see here the release change happened was from 1.5.0->2.0.0, Since we merged the owlbot PR https://github.com/googleapis/google-cloud-php/pull/6452/files, Shouldn't this be 1.6.0?

Reply:
My reasoning was to revert it, since `1.6.0` hasn't been released. If we revert it then release-please will release the new version next time.

But we have two options here - we could also change it to `1.6.0`, and run the release job to create the new release (`1.6.0`) manually. This may be the better option...

Reply:
Sounds good, thank you!